nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/WriteAheadFlowFileRepository.java

```diff
@@ -202,13 +201,12 @@ public void initialize(final ResourceClaimManager claimManager, final Repository
         // delete backup. On restore, if no files exist in partition's directory, would have to check backup directory
         this.serdeFactory = serdeFactory;
 
-        // The specified implementation can be plaintext or encrypted; the only difference is the serde factory
-        if (walImplementation.equals(SEQUENTIAL_ACCESS_WAL) || walImplementation.equals(ENCRYPTED_SEQUENTIAL_ACCESS_WAL)) {
+        try {
             // TODO: May need to instantiate ESAWAL for clarity?
             wal = new SequentialAccessWriteAheadLog<>(flowFileRepositoryPaths.get(0), serdeFactory, this);
-        } else {
+        } catch (final Exception ex) {
```

Comment:
This `catch` seems unnecessary because there is only one implementation of the Write-Ahead Log supported.